Corgibytes.Freshli.Cli/CommandRunners/CacheCommandRunner.cs

```diff
@@ -33,5 +34,37 @@ public override int Run(CachePrepareCommandOptions options)
             return success ? 0 : 1;
         }
     }
-}
 
+    public class CacheDestroyCommandRunner : CommandRunner<CacheDestroyCommandOptions>
+    {
+        public CacheDestroyCommandRunner(IServiceProvider serviceProvider, Runner runner)
+            : base(serviceProvider, runner)
+        {
+
+        }
+
+        public override int Run(CacheDestroyCommandOptions options)
+        {
+            // Skip prompt if the --force flag is passed
+            if (options.Force)
+                goto Destroy;
+
+            // Prompt the user whether they want to destroy the cache
+            Console.Out.Write($"Do you want to destroy the Freshli cache at {options.CacheDir.ToString()}? [y/N] ");
+            string choice = Console.In.ReadLine();
+            string[] choicesToProceed = {"y", "Y"};
```

Comment:
The `var` keyword can be use here.

```suggestion
            var choice = Console.In.ReadLine();
            var choicesToProceed = {"y", "Y"};
```

Reply:
Is there a particular reason why you want to use `var`? Preference? Refactoring?

Personal position, I find explicit types to be more readable unless the type is appearing explicitly somewhere else in the line. In this case, I think `var` reduces readability, or more specifically, increases cognitive load while reading.

Reply:
It's perhaps more of a personal preference after working with languages with more flexible type systems. I like how readable the resulting code is. And if I'm curious about the exact type, there are many different ways to go about getting that information. 

Essentially, I'm a fan of using the `var` keyword everywhere that it can be used. If the compiler is not able to figure out the type, then it lets you know.

Reply:
Noted. With all due respect, since I'm part of the team that will have to keep maintaining this, I really don't want to change this on that basis. I spend more time in Python than anything, and deeply love dynamic typing and duck-typing, but C# (like C++, which has `auto`) isn't that, and types can lead to a lot of surprises. If this comes down to preferences, the cognitive load is lower at least on me (and I know it is on many others) if `var` (like `auto`) is reserved for cases where the type is either explicit elsewhere in the line, or is particularly convoluted.